test/functional/tools/validation_hdf5.xml

```diff
@@ -13,7 +13,8 @@
             <param name="input" value="matrix.cool" />
             <output name="output">
                 <assert_contents>
-                    <has_h5_keys keys="bins, chroms,indexes,pixels" />
```

Comment:
The space was intended to test the stripping.

Reply:
Ahh, that makes sense.